config/postgres/initdb/scripts/create-tables.sql

```diff
@@ -0,0 +1,58 @@
+\c record_expunge;
+
+CREATE EXTENSION "uuid-ossp";
+CREATE EXTENSION citext;
+
+CREATE TABLE users (
+    user_id         UUID NOT NULL,
+    username        CITEXT UNIQUE NOT NULL,
+    email           CITEXT UNIQUE NOT NULL,
+    hashed_password TEXT NOT NULL,
+    password_salt   TEXT NOT NULL,
```

Comment:
If using `bcrypt` to hash the passwords the salt is stored in the hash. Are we planning to use a different hashing function?